selenium/pom.xml

```diff
@@ -211,5 +211,11 @@
       <version>4.9.0</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-remote-driver</artifactId>
+      <version>4.13.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
We don't need to install a new test dependency to get the ChromeDriver's capabilities (see below comment on how to cast it to a typeof ChromeDriver)